tests/h/search/query_test.py

```diff
@@ -3,188 +3,182 @@
 
 import pytest
 
-from h import search
+from h.search import Search, query
 
 
 class TestTopLevelAnnotationsFilter(object):
 
-    def test_it_filters_out_replies_but_leaves_annotations_in(self, Annotation, search):
-        annotation = Annotation(shared=True)
```

Comment:
I know @seanh mentioned something about keeping the shared=True here. For me it's a debate between writing tests that make sense vs writing true unit tests. I would prefer to keep the shared=True out of these unit tests and move them to the integration tests (aka tests that test with multiple filters/aggregators/matchers). Alternatively, we could keep the shared=True here but then I think we should append the shared filter.

Reply:
We discussed this and decided to just leave it as is without the shared=True.